src/ixdat/techniques/ec.py

```diff
@@ -1,18 +1,386 @@
+                corresponding to more than one file).
+            component_measurements (list of Measurements): The measurements of which
+                this measurement is a combination
+            reader (Reader): The file reader (None unless read from a file)
+            plotter (Plotter): The visualization tool for the measurement
+            exporter (Exporter): The exporting tool for the measurement
+            sample (Sample or str): The sample being measured
+            lablog (LabLog): The log entry with e.g. notes taken during the measurement
+            tstamp (float): The nominal starting time of the measurement, used for
+                data selection, visualization, and exporting.
+            ec_technique (str): The electrochemsitry sub-technique
+            RE_vs_RHE (float): The refernce electrode potential on the RHE scale in [V]
+            A_el (float): The electrode area in [cm^2]
+            R_Ohm (float): The ohmic drop resistance in [Ohm]
+        """
+        super().__init__(name, **kwargs)
 
-    def __init__(self, ec_technique=None, **kwargs):
-        super().__init__(**kwargs)
         self.ec_technique = ec_technique
-        self.E_str = "Ewe/V"
-        self.I_str = "I/mA"
+        if RE_vs_RHE or A_el or R_Ohm:
+            self.calibrate(RE_vs_RHE, A_el, R_Ohm)
+        self.plot_vs_potential = self.plotter.plot_vs_potential
+
+    @property
+    def aliases(self):
+        """A dictionary with the names of other data series a given name can refer to"""
+        a = self._aliases.copy()
+        a.update({value: [key] for (key, value) in EC_FANCY_NAMES.items()})
+        return a
+
+    @property
+    def calibrations(self):
+        """The list of calibrations of the measurement.
+
+        The following is necessary to ensure that all EC Calibration parameters are
+        joined in a single calibration when processing. So that "potential" is both
+        calibrated to RHE and ohmic drop corrected, even if the two calibration
+        parameters were added separately.
+        """
+        full_calibration_list = self.calibration_list
+        good_calibration_list = [self.ec_calibration]
+        for calibration in full_calibration_list:
+            if calibration.__class__ is ECCalibration:
+                # Then we have all we need from it
+                continue
+            good_calibration_list.append(calibration)
+        return good_calibration_list
+
+    @property
+    def ec_calibration(self):
+        """A calibration joining the first RE_vs_RHE, A_el, and R_Ohm"""
+        return ECCalibration(RE_vs_RHE=self.RE_vs_RHE, A_el=self.A_el, R_Ohm=self.R_Ohm)
+
+    @property
+    def RE_vs_RHE(self):
+        """The refernce electrode potential on the RHE scale in [V]"""
+        for calibration in self.calibration_list:
+            if hasattr(calibration, "RE_vs_RHE") and calibration.RE_vs_RHE is not None:
```

Comment:
right. The way it is here, it's only the first calibration that counts. Hmmm... if anything it should be the last one! So that the most recent calibration is "trusted". 
Just put in `self.calibration_list.reverse()` in each of these.